geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/DescribeRegionCommand.java

```diff
@@ -333,9 +247,8 @@ private void writeCommonAttributesToTable(TabularResultData table, String attrib
     }
   }
 
-  private boolean writeFixedPartitionAttributesToTable(TabularResultData table,
-      String attributeType, List<FixedPartitionAttributesInfo> fpaList, String member,
-      boolean isMemberNameAdded) {
+  private void writeFixedPartitionAttributesToTable(TabularResultData table,
```

Comment:
In this function, a couple of low-hanging cleanups could be:

(1) Change the big `if` to a guard clause
```
if (fpaList == null) {
    return;
}
```

(2) Change the iterated `while` loop to a `for:` loop.

Also, both of these can be cleanly easily handled by your IDE (although you might need to move the `Iterator<FixedPartitionAttributesInfo> fpaIter = fpaList.iterator();` next to the `while` first).